src/Modal/buildModalCss.ts

```diff
@@ -21,7 +22,17 @@ const baseOverlayCss = (theme) => css`
   top: 0;
 `;
 
-const buildModalCss = ({ theme, overlayCss, modalCss }) => css`
+export type Css = CSSInterpolation | ((theme: Theme) => CSSInterpolation);
```

Comment:
`CSSInterpolation` allows the typical stuff like an object with CSS property key/values, or an emotion `css` construct.